web/src/engine/websites/Akuma_e2e.ts

```diff
@@ -0,0 +1,24 @@
+﻿import { describe } from 'vitest';
+import { TestFixture } from '../../../test/WebsitesFixture';
+
+const fixture = new TestFixture({
+    plugin: {
+        id: 'akuma',
+        title: 'Akuma.moe'
+    },
+    container: {
+        url: 'https://akuma.moe/g/dey7wcbd',
+        id: '/g/dey7wcbd',
+        title: '[Isikenpi] Lethargic Sister Tolerates her Brothers Pranks [English][DarklordMTLs]'
+    },
+    child: {
+        id: '/g/dey7wcbd/1',
+        title: '[Isikenpi] Lethargic Sister Tolerates her Brothers Pranks [English][DarklordMTLs]'
+    },
+    entry: {
+        index: 5,
+        size: 223_038,
+        type: 'image/jpeg'
+    }
+});
+describe(fixture.Name, async () => (await fixture.Connect()).AssertWebsite());
```

Comment:
Website is protected by DDoS Guard for me, is this correctly bypassed in this test?

Reply:
I wont let a test that is not passed uncommented. Ofc its passing ;)